javascripts/main.js

```diff
@@ -1,13 +1,19 @@
 // Add your function at the end. First function is of highest priority.
 var contributorsList = [];
+var thankList = ["Thanks", "Shukran", "Merci", "Sukria", "Danke", "Grazie", "Kamsa Hamnida", "Spasibo", "Obrigado"];
```

Comment:
Can we have a Jekyll file for this so that students can add new languages in ease without having to modify the js files!